extensions/omni_httpc/omni_httpc.c

```diff
@@ -202,8 +209,6 @@ struct request {
   h2o_iovec_t request_body;
   // Response body
   StringInfoData body;
-  // Pointer to a counter of completed requests
-  int *done;
```

Comment:
The rationale for deleting this is described here:
https://github.com/omnigres/omnigres/pull/478/files#r1493106458